contrib/clickclack/update.py

```diff
@@ -0,0 +1 @@
+pattern = "(?!archive\\/)[0-9\\.]*(\\.tar\\.gz)"
```

Comment:
this returns a version that's just .tar.gz:  

```
Checking found version: .tar.gz
```

Reply:
tf?? that shouldnt happen lol

Reply:
OH